templates/settings.html

```diff
@@ -1,41 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
-    <script src="https://cdn.jsdelivr.net/npm/chart.js@3.5.1/dist/chart.min.js"></script>
-    <title>Document</title>
-</head>
-<style>
- 
-</style>
-
-<body class="bg-blue-600">
-    <div class="flex justify-between items-center p-6">
-        {% include 'header.html' %}
-        
-    </div>
-    <div class="flex h-auto justify-center items-center">
-        <div class="bg-white w-full m-6 lg:w-1/2 rounded-lg shadow-lg p-6">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+      <script src="https://cdn.jsdelivr.net/npm/chart.js@3.5.1/dist/chart.min.js"></script>
```

Comment:
Hier wäre es noch gut, wenn du den JS Code über deinen Flask Server auslieferst (also eine lokale installation von Tailwind und chart.js). Ansonsten hast du keine Kontrolle, falls sich an den Bibliotheken etwas ändert und evtl. deine App crashed.
Das kannst du machen, indem du einen Unterordner mit z.B. assets machst und da mit npm ein neues Projekt startest. Dann kannst du dort mit Webpack alles erstellen und laden.
Wie z.B. hier: https://github.com/japrozs/elixir-chat-app/tree/master/assets

Wenn du mit npm dann den Javascript Code baust, wird das alles in ein ouput Ordner gepackt (ist in dem webpack file definiert). Diesen kannst du dann in deinen Server einbauen und von da die Files laden.